packages/yew/src/html/conversion.rs

```diff
@@ -177,6 +177,10 @@ impl_into_prop!(|value: String| -> AttrValue { AttrValue::Rc(Rc::from(value)) })
 impl_into_prop!(|value: Rc<str>| -> AttrValue { AttrValue::Rc(value) });
 impl_into_prop!(|value: VNode| -> Children { Children::new(vec![value]) });
 
+impl_into_prop!(|value: &'static str| -> Key { Key::from(value) });
```

Comment:
This should map all types in `Key`'s `From` implementation.
(maybe in `key_impl_from_to_string!` instead of here.)